core/modules/node/node.pages.inc

```diff
@@ -349,7 +349,7 @@ function node_form($form, &$form_state, Node $node) {
   }
   // If the user doesn't have access to scheduling, default to published.
   elseif ($form['options']['status']['#default_value'] == NODE_SCHEDULED) {
-    $form['options']['status']['#default_value'] = NODE_PUBLISHED;
+    $form['options']['status']['#default_value'] = $node->nid ? $status : NODE_PUBLISHED;
```

Comment:
I'm not sure this line is necessary any more after these changes. The default value should already be accurate when $status is calculated earlier above.

Reply:
Yep, it seems to work without it.

Reply:
@jenlampton in your new PR I think you left it in. I tested it with it in but not with it gone.

Reply:
you're right - force pushed to PR.